test/cli/test_log_runtime_archive_main.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+require "syskit/test/self"
+require "syskit/cli/log_runtime_archive_main"
+
+module Syskit
+    module CLI
+        describe LogRuntimeArchiveMain do
+            before do
+                @archive_dir = make_tmppath
+                @mocked_files_sizes = []
+
+                10.times do |i|
+                    ("/dev/tools/syskit/tmp/datasets" / i.to_s).write(i.to_s)
+                end
+
+                @archiver = LogRuntimeArchiveMain.new(
+                    "/dev/tools/syskit/tmp/datasets",
+                    "/dev/tools/syskit/tmp/archive"
+                )
+            end
+
+            it "test cli" do
+                size_files = [6, 2, 1, 6, 7, 10, 3, 5, 8, 9]
+                mock_files_size(size_files)
+                mock_available_space(0.5)
+                CLI::LogRuntimeArchiveMain.start(
+                    ["/tmp/archive", "/tmp/datasets",
+                     "--free-space-low-limit", "345526079490",
+                     "--free-space-freed-limit", "345526079500"]
```

Comment:
You are using two different paths here (/tmp/.. and /dev/tools/syskit/tmp/dataset). Shouldnt you create a separate tmp path for archive_dir and datasets and then use them?

Reply:
fixed.